cli/_cli.php

```diff
@@ -16,7 +16,7 @@
 Minz_Session::init('FreshRSS', true);
 FreshRSS_Context::initSystem();
 Minz_ExtensionManager::init();
-Minz_Translate::init('en');
+Minz_Translate::init(Minz_Translate::DEFAULT_LANGUAGE);
```

Comment:
Would it be better to have it in a separate PR while this PR is about the API information page?

Reply:
@math-GH See #7933